oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -266,6 +266,7 @@ envoy-metric:
     # to append the version number to the service name.
     # Be careful, when using environment variables to pass this configuration, use single quotes(`''`) to avoid it being evaluated by the shell.
     k8sServiceNameRule: ${K8S_SERVICE_NAME_RULE:"${service.metadata.name}"}
+    enabledMALRules: ${SW_ENVOY_METRIC_MAL_RULES:"envoy"}
```

Comment:
Why need this? Could we disable through `SW_ENVOY_METRIC=-`?

Reply:
> Why need this? Could we disable through `SW_ENVOY_METRIC=-`?

We could disable through `SW_ENVOY_METRIC=-` but I thought this was not for disabling, it shares the same idea as https://github.com/apache/skywalking/issues/5853 , we may need this for the same reason as #5853 , do we?

Reply:
What is this related to the #5853 ? Could you be more clear? That issue is asking for adding group for all services involved in the specific dashboard.
Such as envoy service should include a `group` name in name definition, then they wouldn't show up in other dashboard's service list.

Reply:
:( I pasted the wrong issue number, should be https://github.com/apache/skywalking/issues/5035 and https://github.com/apache/skywalking/issues/5530

Reply:
I think this would be a little different actually. I can't see the chance of multiple rule config files in `metric-receiver`. All those two are only general receiver, you could have data from different sources.

Reply:
OK, removed